clang/unittests/Format/FormatTest.cpp

```diff
@@ -9906,12 +9944,16 @@ TEST_F(FormatTest, ReturnTypeBreakingStyle) {
   verifyFormat("class B {\n"
                "  int f() { return 1; }\n"
                "  int g();\n"
+               "  long\n"
+               "  foooooooooooooooooooooooooooo::baaaaaaaaaaaaaaaaaaaar();\n"
```

Comment:
Delete it because it's not a definition?

Reply:
Yeah, not sure if this example is completely relevant in the tests for these subsequent options.
Though to me, it seems that each of the options has a test with the same block of code. Since there are other lines which aren't definitions in the same test.
Wondering if it is worth keeping it so that the symmetry in this area is preserved?
Though, I don't have strong feelings either way if you think it isn't needed.

Reply:
It's a declaration and supposed to have no break after the type. That's why I suggested that we delete it (and others).

Reply:
Deleted.